modules/guardduty-member-account/versions.tf

```
@@ -0,0 +1,11 @@
+terraform {
+  required_version = ">= 0.15"
+
+  required_providers {
+    aws = {
+      source                = "hashicorp/aws"
+      version               = ">= 4.0"
```

I don't see anything in here requiring 4. We can set this to 3...

```suggestion
      version               = ">= 3.0"
```

---

Updated